### ISSUE TITLE
hsdev fails with GHC 7.10.1

### ISSUE THREAD

**Issue by BethAr** [SublimeHaskell/SublimeHaskell#223]
I'm unable to install `hsdev` after updating to GHC 7.10.1, the reason seems to be Hackage's `ghc-mod` being incompatible with the new GHC. If I clone `ghc-mod` master and install it, it works, but `hsdev` is unable to install against it.

The error when trying to install `hsdev` with the `ghc-mod` currently in master is a bit cryptic: 
```
src\HsDev\Cabal.hs:51:9:
    Non type-variable argument
      in the constraint: Data.String.IsString [a]
    (Use FlexibleContexts to permit this)
    When checking that `cabalDev' has the inferred type
      cabalDev :: forall a.
                  (Eq a, Data.String.IsString [a]) =>
                  [a] -> Bool
    In an equation for `isPackageDb':
        isPackageDb p
          = cabalDev p || cabalSandbox p
          where
              cabalDev dir
                = "packages-" `isPrefixOf` dir && ".conf" `isSuffixOf` dir
              cabalSandbox dir = "-packages.conf.d" `isSuffixOf` dir

src\HsDev\Cabal.hs:52:9:
    Non type-variable argument
      in the constraint: Data.String.IsString [a]
    (Use FlexibleContexts to permit this)
    When checking that `cabalSandbox' has the inferred type
      cabalSandbox :: forall a.
                      (Eq a, Data.String.IsString [a]) =>
                      [a] -> Bool
    In an equation for `isPackageDb':
        isPackageDb p
          = cabalDev p || cabalSandbox p
          where
              cabalDev dir
                = "packages-" `isPrefixOf` dir && ".conf" `isSuffixOf` dir
              cabalSandbox dir = "-packages.conf.d" `isSuffixOf` dir
cabal.exe: Error: some packages failed to install:
hsdev-0.1.3.4 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by mvoidex** [comment]
Hmm, I have ghc-mod from master and hsdev installed. I'll try to get into today, thanks

**Comment by mvoidex** [comment]
I have to fix some issues before releasing 1.4.0, but you can clone `hsdev` master and build 1.4.0

**Comment by mvoidex** [comment]
hsdev is now of 0.1.6.0 version, didn't test it with 7.10.1
feel free to reopen issue
